packages/beacon-node/src/execution/engine/http.ts

```diff
@@ -463,6 +465,17 @@ export class ExecutionEngineHttp implements IExecutionEngine {
     return response.map(deserializeExecutionPayloadBody);
   }
 
+  async getBlobs(_fork: ForkName, versionedHashes: VersionedHashes): Promise<(BlobAndProof | null)[]> {
+    const method = "engine_getBlobsV1";
+    assertReqSizeLimit(versionedHashes.length, 128);
+    const versionedHashesHex = versionedHashes.map(bytesToData);
+    const response = await this.rpc.fetchWithRetries<
```

Comment:
we should probably stop trying to query this api if it failed earlier with specific code that method is not implemented and just return `null`

and since we wanna re-check if the EL is updated, could reset that behavior and retry querying if EL becomes online
https://github.com/ChainSafe/lodestar/blob/068fbae928989295706443887c645d476bb95695/packages/beacon-node/src/execution/engine/http.ts#L496-L501

Reply:
added a mechanism to not query for a while if this fails with method not found